### PR TITLE
Add Centos 8 support

### PR DESCRIPTION
**Pull request by zimmerry**
### Description

Add support for CentOS 8.

[List any existing issues this PR resolves]

The default OpenVPN configuration location has changed from `/etc/openvpn/<type>.conf` to `/etc/openvpn/<type>/<type>.conf` in CentOS 8.

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: recipes/service.rb [hunk 1]
```diff
@@ -21,7 +21,12 @@
 # systemd platforms use an instance service
 case node['platform_family']
 when 'rhel'
-  if node['platform_version'] >= '7'
+  if node['platform_version'] >= '8'
```
Comment (tas50) [inline recipes/service.rb:24]:
```suggestion
  if node['platform_version'].to_i >= 8
```

For compatibility with Chef < 16 you'll want to make sure you compare this as an integer

File: resources/conf.rb [hunk 2]
```diff
@@ -29,7 +29,13 @@
                       "#{new_resource.name}.conf.erb"
                     end
 
-  template [node['openvpn']['fs_prefix'], "/etc/openvpn/#{new_resource.name}.conf"].join do
+  conf_location = if platform_family?('rhel') && node['platform_version'] >= '8'
```
Comment (tas50) [inline resources/conf.rb:32]:
```suggestion
  conf_location = if platform_family?('rhel') && node['platform_version'].to_i >= 8
```

Same here

Comment (ramereth) [inline resources/conf.rb:32]:
```suggestion
  conf_location = if platform_family?('rhel') && node['platform_version'].to_i >= 8
```

**Pull request merged**